pkg/filesystem/subvolume.go

```diff
@@ -317,6 +320,15 @@ func listCephFSSubvolumes(ctx context.Context, clientsets *k8sutil.Clientsets, o
 			}
 		}
 	}
+
+	// After listing, show a concise summary of skipped subvolumes due to not-ready state
+	if len(notReadyErrors) > 0 {
+		logging.Warning("Some subvolumes were not ready and were skipped (pending clone or in-progress):")
+		for _, m := range notReadyErrors {
```

Comment:
```suggestion
		for _, m := range notReadyErrors {
```

Could merge the below warnings?